scalafix/input/src/main/scala/fix/CanBuildFromSrc.scala

```diff
@@ -33,4 +33,11 @@ object CanBuildFromSrc {
     cbf2("")
     ()
   }
+
+  def f2[T, That](implicit cbf: CanBuildFrom[Nothing, T, That]): Foo[T, That] =
```

Comment:
Previously it would only take `CanBuildFrom[Nothing, A, CC[B]]` now it accepts `CanBuildFrom[Nothing, A, B]`, but it won't rewrite `e.to[CC] => cbf.fromSpecific(e)`